packages/jupyter-ai/src/components/settings/model-parameters-input.tsx

```diff
@@ -10,22 +20,36 @@ type ModelParameter = {
   isStatic?: boolean;
 };
 
-type StaticParameterDef = {
-  name: string;
-  type: string;
-  label: string;
+export type ModelParametersInputProps = {
+  modelId?: string | null;
 };
 
-// Add some common fields as static parameters here
-const STATIC_PARAMETERS: StaticParameterDef[] = [
-  { name: 'temperature', type: 'float', label: 'Temperature' },
-  { name: 'api_url', type: 'string', label: 'API URL' },
-  { name: 'max_tokens', type: 'integer', label: 'Max Tokens' }
-];
-
-export function ModelParametersInput(): JSX.Element {
+export function ModelParametersInput(
+  props: ModelParametersInputProps
+): JSX.Element {
+  const [availableParameters, setAvailableParameters] = useState<any>(null);
```

Comment:
Would it be possible to improve type-safety and provide developers that might work on this code later with type hints by defining and using a more precise type for available parameters?

Reply:
@dlqqq